### DISCUSSION TITLE
Event Details (Similar to Apple / Google)

### DISCUSSION THREAD

**Discussion by Ezro** [vkurko/calendar#407] [Q&A]
Are there any examples around for being able to set event details when creating or clicking on the event?

e.g.,
<img width="540" alt="image" src="https://github.com/user-attachments/assets/a34d87bd-a780-4e0d-951f-5c28484428ed" />

**Comment by vkurko** [comment]
Maybe this [demo](https://jsbin.com/xehihumugi/edit?html,output) will be useful to you.
